src/Picasso_FacetGeometry.hpp

```diff
@@ -147,6 +147,18 @@ class FacetGeometry
                             fmax( facets( f, 1, 2 ), facets( f, 2, 2 ) ) ) );
         }
 
+#if KOKKOS_VERSION >= 30700
```

Comment:
No that is the opposite, you want that overload always and guard the one with volatile-qualified argument with `#if KOKKOS_VERSION < 30700`

Reply:
Thanks - should be fixed